src/store/Disk.h

```diff
@@ -102,9 +102,12 @@ class Disk: public Controlled
     char *path;
     int index;          /* This entry's index into the swapDirs array */
     int disker; ///< disker kid id dedicated to this SwapDir or -1
+    bool diskerReady; ///< notified by the disker that it is ready
+    bool diskerIndexed; ///< notified by the disker that it is indexed
     RemovalPolicy *repl;
     int removals;
     int scanned;
+    bool indexed;
```

Comment:
Let's document this new field, especially since its name overlaps with another new field. Our description should make it clear why we need both of these fields (and what the difference is).